CLA.md

```diff
@@ -59,7 +59,7 @@ You must also set your [git e-mail](https://help.github.com/articles/setting-you
 to match this e-mail address as well.
 
 If you've already submitted a PR you can correct your user.name and user.email
-and then use use `git commit --ammend --reset-author` and then `git push` to
+and then use use `git commit --amend --reset-author` and then `git push -f` to
```

Comment:
To keep the style consistent, can we use the `--force` flag instead of `-f`?

Reply:
done